### DISCUSSION TITLE
11straps manual

### DISCUSSION THREAD

**Discussion by matteopedani** [holger1411/11straps#113] [General]
I have many question, may be other  people too, can you help   me. 
Matteo 

#  11straps Manual

###  Install
How to install

### Build proceses
How to build the html pages.

#### dev site
How to build the html pages for develop site.

#### production  site
How to build the html pages for production  site.

### Config files 

Where are the files and who to modify them.

**Comment by holger1411** [comment]
Hik!
Please read the readme file first. Installation, build process(es) etc. are included. Also a quick deploy button to deploy your fork directly from Github to Netlify.

Everything else is located in the /src folder

/img -> image folder
/js -> scripts
/scss -> scss sources to generate CSS 
/includes/layouts -> layouts
/includes/snippets -> reusable code snippets which are not layouts
